src/Components/ClusterRules/ClusterRules.spec.ct.js

```diff
@@ -207,6 +207,16 @@ describe('cluster rules table', () => {
     cy.get(EXPANDABLES).should('have.length', 0);
   });
 
+  it.only('expand one row then sort', () => {
+    cy.get(ROOT).find('#expandable-toggle0').click();
+    cy.get(ROOT)
+      .find('th[data-label=Description]')
+      .find('button')
+      .click()
+      .click();
+    cy.get('table').find('tr').find('td[id="expanded-content11"]');
```

Comment:
The selector points to a component that is rendered *always* (try to play with a table where all rows are collapsed, the selector will still yield you a container) => this assertion doesn't do what you expect (will be always true).